appinfo/info.xml

```diff
@@ -2,10 +2,11 @@
 <info>
     <id>contacts</id>
     <name>Contacts</name>
-    <summary>A contacts app for ownCloud</summary>
+    <summary>Manage and sync your contacts across multiple devices</summary>
```

Comment:
That's what I said but marketing does not want to describe the app technically but the use case.. IMO this is misleading and confusing for sysadmins but I'm more than done discussing this. People want to magically make our apps more than they are because it looks better.